### PR TITLE
feat: APP-427 terrasos footer update

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: APP-427

Update Terrasos footer with latest design.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by flagrede** [comment]
@erikalogie You can have a look at the Terrasos footer redesign here: https://deploy-preview-2532--regen-marketplace.netlify.app/projects/1

I tried to mirror their current footer as much as possible.

File: web-marketplace/src/lib/i18n/locales/es.po [hunk 1]
```diff
@@ -6,10 +6,19 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
```
Comment (blushi) [inline web-marketplace/src/lib/i18n/locales/es.po:6]: should we re-run i18n:translate?

**Review by blushi — APPROVED** [review]
Looks good, pre-approving, just pending es translation and @erikalogie to review

**Comment by erikalogie** [comment]
Looks good, I think that the footer links should open in the same window/tab, with the exception of the social media links. Also, I believe that Facebook is missing and the social links aren't in quite the right order.

It also looks like they have the Terrasos yellow as the selected color for the links, so in our case Habitat Banks should be selected and yellow.

Some small changes on mobile:
Monosnap Terrasos – Inversiones para la Conservación de la Biodiversidad 2024-11-05 12-45-07

**Comment by flagrede** [comment]
@erikalogie, I have handled all your feedback.

**Pull request merged**